### ISSUE TITLE
Incorrect implementation of aria-current for the entries

### ISSUE THREAD

**Issue by akash07k** [SSilence/selfoss#1138]
It seems that there's some incorrect implementation of aria-current in the entry content also.
Though, aria-current should be applied to the entry-title only, it has been applied to the whole entry and it's content along with corresponding action buttons such as share, mark as read etc.
Due to this, screen rearers are reading "current" with every line which is very problematic.
I'm trying to find the issue, but couldn't got any success untill now.

**Comment by jtojnar** [comment]
Fixed in #1141
